tests/src/test_helpers/ssl_helpers.c

```diff
@@ -771,7 +771,6 @@ int mbedtls_test_ssl_endpoint_init(
 
     mbedtls_ssl_init(&(ep->ssl));
     mbedtls_ssl_config_init(&(ep->conf));
-    mbedtls_ssl_conf_rng(&(ep->conf), mbedtls_test_random, NULL);
```

Comment:
I'm adding a comment here because I cannot add it to the proper line of this file. It seems to me that after this PR the function `mbedtls_test_random` won't be used anymore. If so can you please remove it from `ssl_helpers.c` and `ssl_helpers.h`?
Then on `ssl_helpers.c` there is also one call to `srand()` which I think it's connected to the fact that `rand()` was used in `mbedtls_test_random`. Since we're removing that function, can you please also remove the `srand()` call?

Reply:
Ok, so I will set this comment as resolved ;)